### ISSUE TITLE
Multiple roll queries in a single line

### ISSUE THREAD

**Issue by quinnigan** [ArtemGr/Sidekick#14]
I noticed that there was a closed issue for "Ability to roll multiple dice without summing results" which is similar to what my group and I were looking into. We've been using Sidekick for rolling for our Pathfinder campaign lately and some of us have been wondering about rolling multiple attacks in a single go, as many of us are at the point of having multiple attacks.

Being able to do something like the following would be useful, if it was possible to add it. 
 /r 1d20+10 1d20+5  
@User 1d20+10 = (15 + 10) = 25 1d20+5 = (2 + 5) =7

**Comment by ArtemGr** [comment]
Yeah, rolling multiple attacks is something one can see quite often on the table, I wonder how could I've missed that.

I'll think on the syntax.

Meanwhile, maybe the `/r max (1d20+10, 1d20+5)` workaround would help.

**Comment by quinnigan** [comment]
That'll probably be a little easier to do than single rolls. Doesn't do _all_ the math, of course, but it's definitely a workaround! We'll try that out next time we play and see how it goes. :)

**Comment by ArtemGr** [comment]
How about "/r 1d20+10; 1d20+5"?

Using a semicolon as the separator would be much simpler to parse.

**Comment by quinnigan** [comment]
I think that'd be pretty simple and clear!

**Comment by Rimhollow** [comment]
Is this still being planned? It would be terribly useful.

Thanks for the great tool, in any case!
